### PR TITLE
Added CSS handles to payment method & address

### PR DESCRIPTION
**Pull request by razvanudream**
#### What does this PR do? \*

It simply adds some CSS handles on PaymentMethod and Address components in order to style those blocks

#### How to test it? \*

Link this updated version on workspace that has order-placed default implementation and inspect element to check for the new CSS handles

#### Related to / Depends on \*

image
image

### CODE REVIEW

File: react/Address.tsx [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
   pickup?: Parcel
 }
 
+const CSS_HANDLES = [
+  'addressContainer'
+] as const
+
 const Address: FunctionComponent<Props> = ({ address, pickup }) => {
+
+  const handles = useCssHandles(CSS_HANDLES)
```
Comment (igorbrasileiro) [inline react/Address.tsx]:
```suggestion
  const handles = useCssHandles(CSS_HANDLES)
  
```

**Review by igorbrasileiro — APPROVED** [review]
LGTM

File: react/tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
+    "jsx": "react",
```
Comment (salesfelipe) [inline react/tsconfig.json]:
```suggestion
```

File: react/PaymentMethod.tsx [hunk 3]
```diff
@@ -50,11 +56,12 @@ const PaymentMethod: FunctionComponent<Props & InjectedIntlProps> = ({
   const [isOpen, setIsOpen] = useState(false)
   const hasLastDigits = !!payment.lastDigits
   const isBankInvoice = payment.group === 'bankInvoice'
+  const handles = useCssHandles(CSS_HANDLES)
 
   return (
     <article className="flex justify-between">
       <div className="t-body lh-solid">
-        <p className="c-on-base">{paymentGroupSwitch(payment, intl)}</p>
+        <p className={`${handles.paymentType} c-on-base`}>{paymentGroupSwitch(payment, intl)}</p>
```
Comment (salesfelipe) [inline react/PaymentMethod.tsx]:
```suggestion
        <p className={`${handles.paymentGroup} c-on-base`}>{paymentGroupSwitch(payment, intl)}</p>
```

File: react/PaymentMethod.tsx [hunk 4]
```diff
@@ -25,6 +26,11 @@ interface Props {
   currency: string
 }
 
+const CSS_HANDLES = [
+  'paymentType',
```
Comment (salesfelipe) [inline react/PaymentMethod.tsx]:
```suggestion
  'paymentGroup',
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## Added
+
+- Added CSS Handles `paymentType` & `paymentValue` on PaymentMethod
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
- Added CSS Handles `paymentGroup` & `paymentValue` on PaymentMethod
```

File: docs/README.md [hunk 6]
```diff
@@ -313,6 +313,9 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `customerInfoListEmail`   |
 | `customerInfoListDocument`   |
 | `customerInfoListPhone`   |
+| `paymentType`   |
```
Comment (salesfelipe) [inline docs/README.md]:
```suggestion
| `paymentGroup`   |
```

**Review by salesfelipe — CHANGES_REQUESTED** [review]

**Review by salesfelipe — APPROVED** [review]

**Comment by salesfelipe** [comment]
Thanks for your contribution @razvanudream 🙏

**Pull request merged**